src/components/SearchModal/CurrencySearchModal.tsx

```diff
@@ -3,12 +3,19 @@ import React, { useCallback, useEffect, useState } from 'react'
 import useLast from '../../hooks/useLast'
 import { WrappedTokenInfo } from '../../state/lists/wrappedTokenInfo'
 import Modal from '../Modal'
-import { CurrencySearch } from './CurrencySearch'
+import { CurrencySearch } from 'components/SearchModal/CurrencySearch'
 import { ImportToken } from './ImportToken'
 import usePrevious from 'hooks/usePrevious'
 import Manage from './Manage'
 import { TokenList } from '@uniswap/token-lists'
 import { ImportList } from 'components/SearchModal/ImportList'
+import styled from 'styled-components/macro'
+
+export const GpModal = styled(Modal)`
```

Comment:
Same, Mod file?

Reply:
ah yes this should, thought i was in the mod here thanks